custom-script/utils.py

```diff
@@ -13,6 +15,37 @@
 MODEL_INPUTS = ["B02", "B03", "B04", "NDWI", "NDMI"]
 MODEL_INPUTS_STR = ", ".join(MODEL_INPUTS)
 
+FEATURES_SAMPLED = FeatureType.DATA, "FEATURES_SAMPLED"
+IS_DATA_SAMPLED = FeatureType.MASK, "IS_DATA_SAMPLED"
+LABELS_SAMPLED = FeatureType.MASK_TIMELESS, "water_label_SAMPLED"
+
+
+def prepare_data(eopatches, gdf):
```

Comment:
This method would should take as input the train/test array, as it now relies on (hardcoded) columns to be available in geodataframe. Probably similar for eopatch features, which are (hardcoded) in the utils.

Reply:
Outdated moved into notebook.